v1.0/docs/admin/high-availability.md

```diff
@@ -262,7 +262,7 @@ set the `--apiserver` flag to your replicated endpoint.
 
 ## Vagrant up!
 
-We indeed have an initial proof of concept tester for this, which is available [here](../../examples/high-availability/).
+We indeed have an initial proof of concept tester for this, which is available [here](https://releases.k8s.io/v1.0.1/examples/high-availability).
```

Comment:
See branch starting at line 70 in main.go.

Basically this links to a directory without a `readme.md`. This won't have an index.html when the link is followed, so you'll get a 404. In our repo, these links just take you to a directory listing. I figured getting to a version tagged github directory listing was better than 404 in these cases.